aten/src/ATen/native/cuda/MultiTensorApply.cuh

```diff
@@ -38,6 +38,17 @@ __device__ __forceinline__ void load_store(
   ((LT*)dst)[dst_offset] = ((LT*)src)[src_offset];
 }
 
+template <typename dT, typename sT>
+__device__ __forceinline__ void load_store(
```

Comment:
okay this was too naive, aligned_vector probably can't be casted like this. sigggggh